src/JSInterop/Microsoft.JSInterop/test/JSRuntimeExtensionsTest.cs

```diff
@@ -103,7 +103,7 @@ public async Task InvokeAsync_WithTimeout()
                     // it'll get cancelled eventually.
                     Assert.True(cts.CanBeCanceled);
                 })
-                .Returns(new ValueTask<string>(expected));
+                .Returns(new ValueTask<string?>(expected));
```

Comment:
Does Moq not expect the same type in the setup above?

Reply:
`InvokeAsync<string>` returns `ValueTask<string?>`

Reply:
We changed the shape of this API. Where it previously assumed a JSInterop call always never returned a nullable value, now it says it says that it can and this reflects that change.